docs/usage/faq.md

```diff
@@ -41,6 +41,53 @@ If you're self hosting Renovate, use the latest release if possible.
 
 ## What if I need to .. ?
 
+### Tell Renovate to ask for approval before creating a Pull Request
+
+The default behavior is that Renovate creates a pull request right away whenever there's an update.
+But maybe you want Renovate to ask for your approval _before_ it creates a pull request.
+Use the "Dependency Dashboard approval" workflow to get updates for certain packages - or certain types of updates - only after you give approval via the Dependency Dashboard.
+
+The basic idea is that you create a new `packageRules` entry and describe what kind of package, or type of updates you want to approve beforehand.
+
+Say you want to manually approve all major `npm` package manager updates:
```

Comment:
Maybe we should move the Renovate config examples to the `dependencyDashboardApproval` config option? I do like my wording more than what we use over there... 😄 

I'm not sure how much of an config example we want to give in the FAQ? There's a bit of a tradeoff with duplicating configs, or risking the reader not getting the point from just reading the FAQ. Open for suggestions and thoughts on this! 😉